src/EnergyPlus/DataSurfaces.cc

```diff
@@ -734,4 +734,26 @@ Real64 AbsBackSide(EnergyPlusData &state, int SurfNum)
     return AbsorptanceFromExteriorBackSide + AbsorptanceFromInteriorBackSide;
 }
 
+void GetAddOnOverrideSurfaceList(EnergyPlusData &state)
+{
+    for (int surfNum = 1; surfNum <= state.dataSurface->TotSurfaces; ++surfNum) {
+        int materNum = GetMaterialNumFromSurfNum(state, surfNum);
```

Comment:
Breaking this out into its own function is not really helping you here.

Reply:
yeah, I'll just inline the content. Although I'm not sure if this type of lookup operation is performed a lot, whether it would be better to make a helper function like this.